armi/tests/tutorials/data_model.ipynb

```diff
@@ -455,9 +455,7 @@
    "metadata": {},
    "source": [
     "## Loading from the database\n",
-    "Once you have a database, you can load an operator object which has access to the reactor and case settings file.\n",
-    "\n",
-    "Warning: While the reactor is in the same state as it was when we wrote the database, the operator may not be. This is because ARMI uses the parameter system to load state, and only the reactor and its children have parameters. Furthermore, anything not stored in a paramter is not written to the database."
```

Comment:
I feel like there was value in the statement that was removed about anything not stored as a parameter is not in the database, as this applies to block local variables, etc.